.gitmodules

```
@@ -124,6 +124,9 @@
 [submodule "projects/multicenter-phantom"]
 	path = projects/multicenter-phantom
 	url = https://github.com/conpdatasets/multicenter-phantom.git
+[submodule "projects/Bigbrain_3DROIS"]
+	path = projects/Bigbrain_3DROIS
```

```suggestion
	path = projects/Bigbrain_3DROIS
```

but the second file in the file diff of Github is `projects/BigBrain_3DROIs` (lower case `s` vs upper case `S`)